keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -436,7 +449,7 @@ will be allowed to be used at a time.
 
 #### Relationship with Pod.spec.restartPolicy
 
-For Alpha we may limit this feature by disallowing the use of `onExitCodes` when
+For Alpha we limit this feature by disallowing the use of
```

Comment:
What's the decision for Beta?

Reply:
No decision yet. I can see 3 options:
1. Disable the feature for restartPolicy=OnFailure (currently implemented)
2. Enable the feature for restartPolicy=OnFailure, but prevent rules using `onExitCodes` 
3. Enable the feature for restartPolicy=OnFailure and allow for `onExitCodes`.

My preference is for 1. Simple implementations (in which Kubelet is not using pod failure policy rules) of 2 and 3 have the following race conditions:
Kubelet can perform an unnecessary container restart even if the container failed due to exceeding resource limit, matching the FailJob rule using onPodConditions.
(specific to 3): Kubelet can perform an unnecessary container restart even if the exit code of the failed container matches the FailJob rule using onExitCodes.

Reply:
Ok, removed the corresponding point from graduation criteria and updated the "Relationship with Pod.spec.restartPolicy" section.